src/main/kotlin/org/jitsi/nlj/PacketInfo.kt

```diff
@@ -81,6 +81,12 @@ class PacketInfo @JvmOverloads constructor(
      * was an incoming packet and not one created by jvb itself).
      */
     var receivedTime: Long = -1L
+
+    /**
+     * Whether this packet has been recognized to contain only silence.
+     */
+    var silence: Boolean = false
```

Comment:
if it's going to be at the `PacketInfo` level, can we call this something more generic (`discard`/`shouldDiscard`)?

Reply:
okay